src/components/HyperlinkingWrapper.js

```diff
@@ -193,6 +197,17 @@ export default class HyperlinkingWrapper extends Component {
         this.onClose();
     }
 
+    getRangeMinusWhitespace(range) {
+        const selectedText = this.quill.getText(range);
+        const leadingWhitespaceCount = selectedText.length - selectedText.trimLeft().length;
```

Comment:
`trimLeft/Right` will blow on IE without polyfill https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimStart

Reply:
I'd go with regexp solution 
```suggestion
        const leadingWhitespaceCount = selectedText.length - selectedText.replace(/^\s*/, '').length;
```